### DISCUSSION TITLE
Gemini: customize HarmBlockThreshold?

### DISCUSSION THREAD

**Discussion by parseiro** [logancyang/obsidian-copilot#719] [Q&A]
I need to ask my Obsidian vault questions that involve healthcare, but they often get blocked by Gemini.

Is there a way to customize HarmBlockThreshold? Thanks

**Comment by logancyang** [comment]
I encountered this issue randomly too. I tried setting HarmBlockThreshold in code and it's still not working at times. Here's an issue I opened at LangchainJS https://github.com/langchain-ai/langchainjs/issues/7018#issuecomment-2424194011, will check if this helps.

**Reply by logancyang** [reply]
I added all safety settings to `BlockNone`. Let me know if you still get blocked @parseiro
